packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -645,6 +649,93 @@ describe("Garbage Collection Tests", () => {
 				);
 			});
 
+			/**
+			 * Here, the base snapshot contains nodes that have timed out. The test validates that we generate errors
+			 * when these nodes are used via trailing ops. It simulates trailing ops by ensuring that garbage collector
+			 * is not loaded and GC does not run before nodeUsed is called.
```

Comment:
I don't understand what you mean by "ensuring that garbage collector is not loaded".  I thought the point of the last PR (and the `initalizeBaseState` call on line 687) is that GC will be loaded right away.

Reply:
It means GC is not loaded until "nodeUsed" is called. I will udpate the comment to make it clearer.